spec/displaynames.html

```diff
@@ -105,11 +105,11 @@ <h1>Internal slots</h1>
         <li>The value of the field *"language"* must be a Record which must have fields with the names of one of the valid language displays: *"dialect"* and *"standard"*.</li>
         <li>The language display fields under display name type *"language"* should contain Records which must have fields with the names of one of the valid display name styles: *"narrow"*, *"short"*, and *"long"*.</li>
         <li>The value of the fields *"region"*, *"script"*, *"currency"*, *"calendar"*, and *"dateTimeField"* must be Records, which must have fields with the names of all display name styles: *"narrow"*, *"short"*, and *"long"*.</li>
-        <li>The display name style fields under display name type *"language"* should contain Records with keys corresponding to language codes matching the `unicode_language_id` production. The value of these fields must be string values.</li>
+        <li>The display name style fields under display name type *"language"* should contain Records with keys corresponding to language codes that can be matched by the <code>unicode_language_id</code> language tag nonterminal. The value of these fields must be string values.</li>
         <li>The display name style fields under display name type *"region"* should contain Records with keys corresponding to region codes. The value of these fields must be string values.</li>
         <li>The display name style fields under display name type *"script"* should contain Records with keys corresponding to script codes. The value of these fields must be string values.</li>
         <li>The display name style fields under display name type *"currency"* should contain Records with keys corresponding to currency codes. The value of these fields must be string values.</li>
-        <li>The display name style fields under display name type *"calendar"* should contain Records with keys corresponding to a String value with the `type` given in Unicode Technical Standard 35 for the calendar used for formatting. The value of these fields must be string values.</li>
+        <li>The display name style fields under display name type *"calendar"* should contain Records with keys corresponding to calendar identifiers that can be matched by the <code>type</code> language tag nonterminal. The value of these fields must be string values.</li>
```

Comment:
How about "Records with keys corresponding to calendar identifiers that can be matched by the <code>type</code> nonterminal defined in Unicode Technical Standard 35."